### ISSUE TITLE
S: Analysis of buildings in proposed UES Special District for MN Borough Office

### ISSUE THREAD

**Issue by lseirup** [NYCPlanning/edm-overview#226]
Proposed Special District would stretch from Third Avenue to York Avenue, from 59th to 96th street, and include include both sides of Third Avenue and York Avenue to the outer edge of their zoning districts, which is 100 feet into the midblock. Tony Lechuga wants to identify buildings meeting the following criteria:

1. Built FAR between 10 and 12
2. Maximum building height – 210 feet
3. 85 feet high before any setbacks

**Comment by lseirup** [comment]
Project folder: G:\EDM\GIS Projects\20190819 MN borough UES analysis (G: drive is for GSS)
Methodology:

1. Created polygon covering Third Ave to York, 59th to 96th. 
2. Buffered polygon by 100 ft, then clipped off 100 foot buffer at 59th and 96th.
3. Selected all lots in MapPLUTO 19v1 that intersect with the proposed SD boundaries, marking those that were split by the boundary
4. Set a flag for lots meeting FAR criterium
5. Selected building footprints in the proposed boundaries with HEIGHTROOF >= 210 feet
6. Reviewed building footprints without a HEIGHTROOF value and added to selection if warranted.
7. Joined footprints to lots and updated the lot dataset with HEIGHTROOF info and a flag indicating the lot met 2nd criterium. Checked lots meeting both criteria that had more than one building, verifying that we captured height of the tallest building.

Sent map and spreadsheet with 57 lots to Tony. Suggested Cyclomedia as a way to look at setbacks.
